app/views/notes/_comments.html.erb

```diff
@@ -17,9 +17,9 @@
     <% if current_user %>
       <%= render :partial => "comments/form", :locals => { title: I18n.t('notes._comments.post_comment'), comment: false, placeholder: I18n.t('notes._comments.post_placeholder') } %>
     <% else %>
-      <p><a href="#" data-toggle="modal" data-target="#loginModal">
-        <%= translation('layout._header.login.login_title') %> </a> to comment.
-      </p>
+      <p>
+      <a href="#" data-toggle="modal" data-target="#loginModal">
+      <%= link_to translation('layout._header.login.login_title'), "/login?return_to=#{request.path}" %> to comment
```

Comment:
Hey! There are two links here, this section should stay the same as before like this:
```html
      <p><a href="#" data-toggle="modal" data-target="#loginModal">
        <%= translation('layout._header.login.login_title') %> </a> to comment.
      </p>
```

Reply:
:+1: